### ISSUE TITLE
Statut non mis à jour immédiatement après suppression d'une procédure opposable

### ISSUE THREAD

**Issue by HermanceGauth** [MTES-MCT/Docurba#656]
Cas de Morlaix : https://docurba.beta.gouv.fr/ddt/29/collectivites/29151/commune
Le statut du PLUiH devrait être opposable et non précédent.
En lien avec : https://github.com/MTES-MCT/Docurba/issues/143

**Comment by HermanceGauth** [comment]
Cela a perturbé l'utilisateur qui a créé une nouvelle procédure principale au lieu d'une nouvelle procédure secondaire. Le fait que les statut ne se mettent pas correctement à jour induit des erreurs de la part des utilisateurs que nous devront ensuite gérer à la main ou alors leur demander de tout refaire ce qui est vraiment gênant.

**Comment by UngererFabien** [comment]
J'ai l'impression que ce cas à été géré par le daily dump. Est ce qu'on a d'autres exemples ?

**Comment by HermanceGauth** [comment]
Yes le statut est bon pour le premier PLUiH le problème maintenant c'est qu'on a deux procédures en cours j'ai vu avec @lerayj pour supprimer celle qui est en trop mais si une vient de Sudocuh, elle va réapparaître à chaque dump donc il faudrait que je dise à la DDT de supprimer directement dans Sudocuh?

**Comment by HermanceGauth** [comment]
@UngererFabien à la suppression d'une procédure opposable, le précédent ne repasse pas opposable automatiquement, j'ai dû repasser par la pre-prod et actualiser manuellement pour avoir le bon statut.

**Comment by HermanceGauth** [comment]
Sera vu avec la revue de l'import du dump et l'issue 814.

**Comment by HermanceGauth** [comment]
@UngererFabien j'avais noté que cette issue serait vu avec les remarques de Claire dans la 814, je viens de retester en dev, j'ai le même comportement que le 16/09.

**Comment by HermanceGauth** [comment]
Regarder pourquoi le déclenchement ne se fait pas (webhook)

**Comment by HermanceGauth** [comment]
A regarder en même temps que les statuts SCoT

**Comment by HermanceGauth** [comment]
PR prévue le 16/12

**Comment by UngererFabien** [comment]
@rik je t'ai fais une première PR pour la partie back API: https://github.com/betagouv/docurba-nuxt3/pull/6

**Comment by UngererFabien** [comment]
![Image](https://github.com/user-attachments/assets/dd271f75-131a-4223-ba2f-c77f1936ccba)

J'ai mis en place l'automatisme. Mais ce n'est pas instantané. Du coup j'ai rajouter un loader et un message d'alerte quand il y a eu un delete recent.

**Comment by HermanceGauth** [comment]
Testé ce jour sur dev et prod, le statut apparaît bon en rechargeant deux fois la page. Pas idéal mais ok pour le moment.

**Comment by rik** [comment]
Le [code censé amélioré cela](https://github.com/betagouv/docurba-nuxt3/pull/6) n'est pas sur les serveurs donc je ne comprends pas :
- Comment @UngererFabien peut avoir mis un "automatisme" en place. Il va me falloir plus de détails.
- Comment @HermanceGauth peut constater les effets de quelque chose qui n'est pas en production.

Je repousse à Iteration 19 pour qu'on puisse en discuter.

**Comment by rik** [comment]
Le code est effectivement en production mais utilise la version Vercel du site.

**Comment by rik** [comment]
On va faire demi-tour sur ce sujet au profit de #1107.

**Comment by rik** [comment]
Je passe ce sujet en post migration car cela sera instantané en normalisant la base de données.
